File: src/main/java/models/Commande.java
package models;

import java.time.LocalDate;
import java.util.Date;
import java.time.LocalDate;

public class Commande {

    private int idC, numTel;

    private String adresseLivraison;





    // Constructeur par défaut
    public Commande() {
    }

    // Constructeur avec tous les attributs
    public Commande(int idC, String adresseLivraison, int numTel) {
        this.idC = idC;
        this.numTel = numTel;
        this.adresseLivraison = adresseLivraison;

    }

    public Commande(int numTel, String adresseLivraison) {
        this.numTel = numTel;
        this.adresseLivraison = adresseLivraison;

    }

    // Getters et setters


    public int getIdC() {
        return idC;
    }

    public void setIdC(int idC) {
        this.idC = idC;
    }

    public int getNumTel() {
        return numTel;
    }

    public void setNumTel(int numTel) {
        this.numTel = numTel;
    }

    public String getAdresseLivraison() {
        return adresseLivraison;
    }

    public void setAdresseLivraison(String adresseLivraison) {
        this.adresseLivraison = adresseLivraison;
    }



    @Override
    public String toString() {
        return "Commande{" +
                "idC=" + idC +
                ", numTel=" + numTel +
                ", adresseLivraison='" + adresseLivraison + '\'' +

                '}';
    }
}

File: src/Entity/Commande.php
<?php

namespace App\Entity;

use App\Repository\CategorieRepository;
use Symfony\Component\Validator\Constraints as Assert;
use Doctrine\ORM\Mapping as ORM;
use App\Repository\CommandeRepository;

#[ORM\Entity(repositoryClass: CommandeRepository::class)]

class Commande
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private $id;

    #[ORM\Column(length:255)]
    #[Assert\NotBlank(message: "Name is required !")]
    private $adresselivraison;


    #[ORM\Column]
    #[Assert\Positive]
    private $numTel;

    #[ORM\ManyToOne(targetEntity: Livreur::class)]
    #[ORM\JoinColumn(name: "id", referencedColumnName: "id")]

    private ?Livreur $livreur;

    // Getters and setters

    public function getIdC(): ?int
    {
        return $this->idC;
    }

    public function getAdresseLivraison(): ?string
    {
        return $this->adresseLivraison;
    }

    public function setAdresseLivraison(?string $adresseLivraison): self
    {
        $this->adresseLivraison = $adresseLivraison;
        return $this;
    }

    public function getNumTel(): ?string
    {
        return $this->numTel;
    }

    public function setNumTel(string $numTel): self
    {
        $this->numTel = $numTel;
        return $this;
    }

    public function getLivreur(): ?Livreur
    {
        return $this->livreur;
    }

    public function setLivreur(?Livreur $livreur): void
    {
        $this->livreur = $livreur;
    }


}